CHANGELOG.md

```diff
@@ -137,6 +137,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix Segment Replication ShardLockObtainFailedException bug during index corruption ([10370](https://github.com/opensearch-project/OpenSearch/pull/10370))
 - Fix some test methods in SimulatePipelineRequestParsingTests never run and fix test failure ([#10496](https://github.com/opensearch-project/OpenSearch/pull/10496))
 - Fix passing wrong parameter when calling newConfigurationException() in DotExpanderProcessor ([#10737](https://github.com/opensearch-project/OpenSearch/pull/10737))
+- Fix SuggestSearch.testSkipDuplicates by forceing refresh when indexing its test documents ([#11068](https://github.com/opensearch-project/OpenSearch/pull/11068))
```

Comment:
nit - I don't think we should include a changelog entry here given this is not a user facing change.

Reply:
On line 138 lines above there is a similar changelog entry to mine.  I think there is still an issue of 'who' exactly is reading these entries, happy to dialog in an issue if you'd like to change the standard.

Since you used the term `nit` I'm going to resolve this without confirmation - thanks.